### ISSUE TITLE
Issue with k8s.io/docs/getting-started-guides/ubuntu/

### ISSUE THREAD

**Issue by ht512512** [kubernetes/kubernetes.github.io#4065]
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**
sudo snap install conjure-up --classic
suggested by the doc, results in 
error: unknown flag `classic'
on Ubuntu 16.04

So tried without the flag --classic
sudo snap install conjure-up
which resulted in
error: cannot perform the following tasks:
- Mount snap "conjure-up" (352) (info failed to parse: invalid confinement type: "classic")
(Not a kubernetes bug, just the documentation correction required)

Basically the documented installation step does not work.
**Proposed Solution:**

**Page to Update:**
http://kubernetes.io/...

**Comment by zacharysarah** [comment]
@ht512512 Are you still experiencing this issue? 

I encourage you to bring your question to the `#kubernetes-users` channel in [Kubernetes slack](http://slack.k8s.io/).

**Comment by ht512512** [comment]
@zacharysarah 
Thanks.  I went with Kops approach and everything is fine now. Closing this.
